cli/command_logs_test.go

```diff
@@ -90,14 +90,24 @@ func TestLogsMaintenance(t *testing.T) {
 	e.RunAndExpectSuccess(t, "maintenance", "set", "--max-retained-log-count=2")
 	e.RunAndVerifyOutputLineCount(t, 5, "logs", "list")
 
-	e.RunAndExpectSuccess(t, "maintenance", "run")
+	e.RunAndExpectSuccess(t, "maintenance", "run", "--full")
 	e.RunAndVerifyOutputLineCount(t, 3, "logs", "list")
 
 	e.RunAndExpectSuccess(t, "maintenance", "set", "--max-retained-log-age=1ms")
 	e.RunAndVerifyOutputLineCount(t, 4, "logs", "list")
 
-	e.RunAndExpectSuccess(t, "maintenance", "run")
+	e.RunAndExpectSuccess(t, "maintenance", "run", "--full")
 	e.RunAndVerifyOutputLineCount(t, 1, "logs", "list")
+}
+
+func TestLogsMaintenanceSet(t *testing.T) {
```

Comment:
This change is unrelated to the primary purpose of the PR.
It simply splits this check to its own test.